### PR TITLE
feat: add simple cli interface

### PR DESCRIPTION
**Pull request by mstruebing**
Adds a simple CLI interface to manage sources and plugins.

### CODE REVIEW

File: cmd/plugin/plugin.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+func AddCommand(out output.Output) *cobra.Command {
+	return &cobra.Command{
+		Use:   "add",
+		Short: "Adds a plugin",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			out.V(6).Info(fmt.Sprintf("args: %v", args))
+			return nil
+		},
+	}
+}
```
Comment (aweris) [inline cmd/plugin/plugin.go:51]: [question] do we need this one as well, what is the difference between this and `install`

File: cmd/source/source.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+func RemoveCommand(out output.Output) *cobra.Command {
+	return &cobra.Command{
+		Use:   "remove",
+		Short: "Removes a source",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			out.V(6).Info(fmt.Sprintf("args: %v", args))
+			return nil
+		},
+	}
+}
```
Comment (aweris) [inline cmd/source/source.go:50]: We could skip these for now since we'll install it by default. What do you think?

Comment (mstruebing) [reply cmd/source/source.go:50]: I will remove it.

File: cmd/plugin/plugin.go (hunk 1 above)
Comment (mstruebing) [reply cmd/plugin/plugin.go:51]: Depends, this is for adding a plugin to the source and the install is to install a specific version of that plugins tool.
I'm not sure we can manage this by only one command, i.e. `install go 1.19` would need to install the go plugin to asdf in the current setup, but there are a lot of plugins out there and we would need to hardcode all of them

Comment (mstruebing) [reply cmd/plugin/plugin.go:51]: Do you think one command is enough for now?

**Review by aweris — APPROVED** [review]

**Pull request merged**